tests/v2/validation/auth/openldap.go

```diff
@@ -0,0 +1,43 @@
+package auth
+
+import (
+	"context"
+	"time"
+
+	"github.com/rancher/shepherd/clients/rancher"
+	v3 "github.com/rancher/shepherd/clients/rancher/generated/management/v3"
+	kwait "k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	passwordSecretID       = "cattle-global-data/openldapconfig-serviceaccountpassword"
+	authProviderAnnotation = "management.cattle.io/auth-provider-cleanup"
+	locked                 = "rancher-locked"
```

Comment:
nit: Can we call this something like authProvCleanupAnnotationValLocked (I know it seems quite long). But i think this annotation has two possible values - unlocked and rancher-locked. So can we name them appropriately?

Reply:
Addressed